src/game_state/Level.h

```diff
@@ -0,0 +1,24 @@
+#ifndef LEVEL_H_
+#define LEVEL_H_
+
+#include <memory>
+#include <SFML/Audio.hpp>
+
+#include "GameObject.h"
+#include "Platform.h"
+
+class Level : public GameObject {
+ public:
+  Level(std::shared_ptr<sf::Music> music,
+        std::shared_ptr<std::vector<Platform>> level);
+  ~Level();
+
+  std::shared_ptr<sf::Music> getMusic() { return music; }
+  std::shared_ptr<std::vector<Platform>> getLevel() { return level; }
+
+ private:
+  std::shared_ptr<sf::Music> music;
+  std::shared_ptr<std::vector<Platform>> level;
```

Comment:
how about we have a level and a vector of objects in the game state, the level should have more stuff but we might also want to have stuff that is not necessarily a part of the level